types/config.d.ts

```diff
@@ -15,18 +15,14 @@ declare global {
        */
       export interface Json {
         extends?: 'lighthouse:default' | 'lighthouse:full' | string | boolean;
-        settings?: SettingsJson;
+        settings?: SharedFlagsSettings;
```

Comment:
seems like we could just call this settings and CliFlags extends settings then :)

Reply:
> seems like we could just call this settings and CliFlags extends settings then :)

ha, we could. For whatever reason the name `SharedFlagsSettings` does a poor job to indicate that it's literally "the properties shared by flags and settings" to me :)